src/components/layout/Nav.jsx

```
@@ -145,6 +149,13 @@ const Nav = ({ handleSearch }) => {
     </Menu>
   );
 
+  const signOutUser = function() {
+    firebase.auth().signOut().then(function() {
+      goTo("/");
+    }).catch(function() {
```

Empty catch. I'd say at least add a console.error or just remove it.